lib/stripe/stripe_object.rb

```diff
@@ -262,9 +262,10 @@ def method_missing(name, *args)
       # TODO: only allow setting in updateable classes.
       if name.to_s.end_with?('=')
         attr = name.to_s[0...-1].to_sym
+        val = args.first
```

Comment:
I feel a little uneasy about this. Can we rely on the first in the splatted arglist to always be the value? I suppose the only case where we couldn't would be non-boolean, but detail isn't particularly relevant in the context of this method definition.

I had also considered `val, * = args`, but I don't think that's any improvement (and perhaps its harder to grok)